converter/internal/test_common/testing.go

```diff
@@ -86,12 +87,17 @@ func getExpectedDiags(t *testing.T, diagsFile string) []string {
 	if _, err := os.Stat(diagsFile); err == nil {
 		errorBytes, err := os.ReadFile(diagsFile)
 		require.NoError(t, err)
-		errorsString := string(normalizeLineEndings(errorBytes))
```

Comment:
This has been removed from the new code because `bufio.NewScanner` gracefully handles \r\n and \n out of the box.